storage/scratchpad/src/sparse_merkle/test_utils/proptest_helpers.rs

```diff
@@ -33,18 +40,25 @@ pub fn arb_smt_correctness_case() -> impl Strategy<Value = Vec<Action>> {
                     // txns
                     vec(
                         // txn updates
-                        (any::<Index>(), any::<Vec<u8>>()),
-                        1..4,
+                        (
+                            any::<Index>(),
+                            prop_oneof![
+                                any::<Vec<u8>>().prop_map(Op::Delete),
+                                any::<Vec<u8>>().prop_map(Op::Write)
+                            ]
```

Comment:
`any::<Option<Vec<u8>>>()` will do?

Reply:
then it's super hard to generate an existing key for deletion test.